apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -951,7 +931,7 @@ public void loadNonBeneficiaryWhenInsertAndFilterOnExpectRecordLoaded() {
             StaticRifResourceGroup.SAMPLE_A.getResources());
     loadSample(
         "non-Bene sample",
-        CcwRifLoadTestUtils.getLoadOptions(USE_INSERT_IDEMPOTENT_STRATEGY, FILTER_MODE_DISABLED),
```

Comment:
FILTER_MODE_DISABLED should be removed a few lines above this.

Reply:
done (along with renaming and/or removing tests particular to REF_YR counts and skipped records.